python/ray/serve/_private/config.py

```diff
@@ -147,6 +155,20 @@ def user_config_json_serializable(cls, v):
 
         return v
 
+    @validator("logging_config", always=True)
+    def logging_config_valid(cls, v):
+        if v is None:
+            return v
+        if not isinstance(v, dict):
+            raise TypeError(
+                f"Got invalid type '{type(v)}' for logging_config. "
+                "Expected a dictionary."
+            )
+        if "log_level" not in v:
+            v["log_level"] = "INFO"
```

Comment:
why do we need this instead of just setting the default in the schema?

Reply:
the problem is that when are doing "unset=True" in https://github.com/ray-project/ray/blob/master/python/ray/serve/_private/application_state.py#L956

Even though we have default value, we are not able to pass into the deployment status info. So I make it if no log_level is set, set it "INFO" to make sure we always pass the correct value down to the replica. Otherwise it will becomes empty string which is not valid for configuring the logging config.